vite.config.js

```diff
@@ -1,11 +1,12 @@
 import { defineConfig } from 'vite';
 import react from '@vitejs/plugin-react';
 
-// https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
-  port: 3000,
-  
+  base: import.meta.env.DEV ? '/' : '/SuperMario/',
```

Comment:
- [ ] This was creating an error while running npm dev until I deleted it, kindly check it again
![image](https://github.com/richie1988/SuperMario/assets/66097371/2e54b604-208e-4c48-8764-f393dc83c5b2)